geode-core/src/main/java/org/apache/geode/internal/cache/DiskStoreImpl.java

```diff
@@ -3217,14 +3217,16 @@ public void forceIFCompaction() {
     @Override
     public String cancelInProgress() {
       if (isClosed()) {
-        return "The disk store is closed.";
+        return "The disk store is closed";
       } else {
         return null;
       }
     }
 
     @Override
     public RuntimeException generateCancelledException(Throwable e) {
+      logger.info("DiskStoreImpl.isClosed(): " + isClosed() + " isClosing(): " + isClosing());
```

Comment:
this looks like it might be debug logging that should be removed. It does not look like a log message that would be helpful to customers